docs/mdbook/src/android/annotation-guide.md

```diff
@@ -0,0 +1,88 @@
+# Annotation: Marker
+
+This guide will show you how to add Markers in the map.
+
+`Annotation` is an overlay on top of a Map. In package
+`com.mapbox.mapboxsdk.annotations`, it has the following subclasses:
+1. [Marker]
+2. [Polyline]
+3. [Polygon]
+
+A Marker shows an icon image at a geographical location. By default, marker uses
+a provided image as its icon (User may find it in drawable folder from the package
+`com.comapbox.mapboxsdk`).
```

Comment:
This does not look correct.

Reply:
`com.comapbox.mapboxsdk` -> `com.mapbox.mapboxsdk`

Reply:
Ah... it is a little confusing...
User can reference icon drawable with the following code:
```kotlin
com.mapbox.mapboxsdk.R.drawable.maplibre_marker_icon_default
```
But the package name on maven is `org.maplibre.gl:android-sdk`

In 6c18b0e, I just use hyperlink to point out the default marker image.